docs/docs/docs/01-core/openapi/02-architecture.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Architecture
+slug: /openapi/architecture
+tags:
+  - openapi
+  - architecture
+  - components
+---
+
+# Open API
+
+This section explores the overall architecture and main components of the OpenAPI package.
+
+---
+
+## Overview
+
+The package is organized into two main domains:
+
+### The Route Collection
+
+This domain provides objects that extract different sets of routes from a Strapi application.
+
+You can find related files in `src/routes`.
+
+::::info
+This domain could potentially be moved to the Strapi core or Strapi utils in the future since it doesn't have any direct dependencies on OpenAPI itself.
+::::
```

Comment:
```suggestion
:::info
This domain could potentially be moved to the Strapi core or Strapi utils in the future since it doesn't have any direct dependencies on OpenAPI itself.
:::
```